cdap-common/src/main/java/co/cask/cdap/common/kerberos/NoOpOwnerAdmin.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.common.kerberos;
+
+import co.cask.cdap.common.AlreadyExistsException;
+import co.cask.cdap.proto.id.KerberosPrincipalId;
+import co.cask.cdap.proto.id.NamespacedEntityId;
+
+import java.io.IOException;
+import javax.annotation.Nullable;
+
+/**
+ * No-op implementation of {@link OwnerAdmin}.
+ */
+public class NoOpOwnerAdmin implements OwnerAdmin {
```

Comment:
will this work everywhere? In a test, when you deploy an app, then redeploy it, the app fabric will validate that the provided owner is the same as the stored existing owner. Won't that fail then?